src/en/credentials.md

```diff
@@ -0,0 +1,134 @@
+Title: Juju credentials
+
+# Cloud credentials
+
+In order to access your cloud (with the exception of [LXD][lxd]), Juju will
+need to know how to authenticate itself. We use the term “credential” to 
+describe the tokens or keys or secrets needed to spin up resources on clouds.
+
+
+Currently, Juju can use one of three ways to get your credentials for a cloud:
+
+ - Scanning existing credentials (e.g. environment variables, .novarc files)
+ - Importing a user-provided [YAML-formatted][yaml] file.
+ - Entering credentials interactively on the command-line
+
+These methods are explained in more detail below.
+
+### Scanning existing credentials
+
+Some cloud providers (e.g. AWS, Openstack) have commandline tools which rely on 
```

Comment:
This is what I see:

grep -ri commandline | wc -l
3

$ grep -ri 'command line' | wc -l
32